local_planner/src/nodes/local_planner_node.h

```diff
@@ -53,6 +55,33 @@ struct cameraData {
   bool received_;
 };
 
+
+/**
+* @brief struct to contain the parameters needed for the model based trajectory planning
+* when MPC_AUTO_MODE is set to 1 (default) then all members are used for the jerk limited 
+* trajectory on the flight controller side
+* when MPC_AUTO_MODE is set to 0, only up_accl, down_accl, xy_acc are used on the 
+* flight controller side 
+**/
+struct ModelParameters {
+  // clang-format off
+  int mpc_auto_mode = 1; // Auto sub-mode - 0: default line tracking, 1 jerk-limited trajectory
+  float jerk_min = 8.0f; // Velocity-based jerk limit 
+  float up_acc = 10.0f;   // Maximum vertical acceleration in velocity controlled modes upward
+  float up_vel = 3.0f;   // Maximum vertical ascent velocity
+  float down_acc = 10.0f; // Maximum vertical acceleration in velocity controlled modes down
+  float down_vel = 1.0f; // Maximum vertical descent velocity
+  float xy_acc = 5.0f;  // Maximum horizontal acceleration for auto mode and
+                      // maximum deceleration for manual mode
+  float xy_vel = 1.0f;   // Desired horizontal velocity in mission
+  float takeoff_speed = 1.0f; // Takeoff climb rate
+  float land_speed = 0.7f;   // Landing descend rate
+  // limitations given by sensors
+  float distance_sensor_max_height = 5.0f;
```

Comment:
This is mixing concerns, we shouldn't care about distance sensors inside of the simulator, just the limitations. We can leave this in, but it probably won't be used.